client/src/components/pages/jobs/Jobs.css

```diff
@@ -0,0 +1,14 @@
+.mainContainer {
+    padding: 2rem;
+  }
+  
+  .header {
+    color: black;
+  }
+  
+  
+  .cards {
+    display: flex;
+    flex-wrap: wrap;
+  }
+  
```

Comment:
could you make this file `Jobs.module.css`?

Reply:
ok i did it :)